IslandLanding/IslandLanding/ViewModel/GamerTagViewModel.cs

```diff
@@ -29,10 +29,10 @@ public string PopupText
     public GamerTagViewModel()
     {
       SaveCommand = new Command(SaveCommandExcute);
+      InfoViewModel.Play();
```

Comment:
Don't make 'Play()' a static function that's called like this. 
I would recommend creating an AudioService that controls pause and play and also stores the state of whether the audio is playing or not

Reply:
i stored the state in the prefrences so i can check it ,